clash-lib/src/Clash/Netlist/Id/Common.hs

```diff
@@ -95,7 +95,7 @@ prettyName t = maybe t (uncurry (<>)) (maybeTuple t)
 maybeTuple :: Text -> Maybe (Text, Text)
 maybeTuple "(# #)" = Just ("Unit", "")
 maybeTuple "()" = Just ("Unit", "")
-maybeTuple t = first (\n -> "Tup" <> showt n) <$> parseTuple t
+maybeTuple t = first (\n -> "Tuple" <> showt n) <$> parseTuple t
```

Comment:
Is this a cosmetic change, or something that's actually needed? Not opposed to it either way.

Reply:
So in GHC 9.8.1, we no longer have as the original tuple type constructor names `(,)`, `(,,)`,  `(,,,)`, etc. but instead we have `Tuple2`, `Tuple3, `Tuple4` etc.

The above code translated `(,)` to `Tup2`.

Now, in order for all of our code to produce the same output regardless whether were in an environment with GHC 9.8 or a version older than that, we have Clash translate `(,)` to `Tuple2` when we're on an older GHC versions.

Hope that makes sense.

Reply:
Yes, makes sense, thanks